### PR TITLE
Updated version and contents of changelog.MD

### PR DESCRIPTION
**Pull request by saloneerege**

### CODE REVIEW

**Comment by cgriggs01** [comment]
Looks good to me!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
 
-## 1.0.0 (Unreleased)
-
-(https://github.com/vmware/terraform-provider-vmc/tree/HEAD)
-
-[Full Changelog](https://github.com/vmware/terraform-provider-vmc/compare/87cfcf8f21600ef6198389c569d1e988ca30b5a9...HEAD)
```
Comment (dx-sdk-iac) [inline CHANGELOG.md]: Why did you remove the link to full changelog?

**Comment by saloneerege** [comment]
> Looks good to me!

@cgriggs01 how do we reference the PR that have been linked to each enhancement / bug fix ?

**Comment by cgriggs01** [comment]
@saloneerege,  You can build the content of the changelog however you'd like, including links, the only requirements from our side is the version number and tag on the first line of the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
 
 ENHANCEMENTS:
 
-* resource/vmc_sddc: Added nsxt_reverse_proxy_url to SDDC resource data. [\#23](https://github.com/vmware/terraform-provider-vmc/pull/23)
-* data_source/vmc_connected_accounts: Added filtering to return AWS account ID associated with the account number provided in configuration. [\#30](https://github.com/vmware/terraform-provider-vmc/pull/30)
-* resource/vmc_public_ip: Added public IP resource. [\#43](https://github.com/vmware/terraform-provider-vmc/pull/43)
+* vmc_sddc: Added nsxt_reverse_proxy_url to SDDC resource data. [\#23](https://github.com/vmware/terraform-provider-vmc/pull/23)
+* vmc_connected_accounts: Added filtering to return AWS account ID associated with the account number provided in configuration. [\#30](https://github.com/vmware/terraform-provider-vmc/pull/30)
+* provider.go: Added org_id as a required parameter in terraform schema. [\#38](https://github.com/vmware/terraform-provider-vmc/pull/38)
+* data_source_vmc_customer_subnets.go : Adding validateFunctions for sddc and customer subnet resources. [\#41](https://github.com/vmware/terraform-provider-vmc/pull/41)
```
Comment (shylasrinivas) [inline CHANGELOG.md]: Adding to added

Comment (shylasrinivas) [reply CHANGELOG.md]: Should we mention changes made for terraform v0.12 compatibility?

Comment (cgriggs01) [reply CHANGELOG.md]: The v0.12 compatibility should be implied, but if you think i might effect uses, it should be included.

Comment (shylasrinivas) [reply CHANGELOG.md]: Sounds good, we can leave it out.

**Review by shylasrinivas — APPROVED** [review]
